docs/home.md

```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
+The Treasury Account is a Programme Derived Account (PDA) that is controlled by the Treasury Controller program; this means that only the Treasury Controller program can move the funds in the Treasury Account.
 
-### What is gSOL?
+When you deposit your SOL, you will also receive an equivalent amount of gSOL in your wallet.
+You can either keep your gSOL in your wallet, or use it on platforms that support it.
 
-gSOL is a synthetic derivative of SOL, which is minted by the Sunrise Stake protocol
-when you stake SOL. It is a 1:1 representation of SOL, and can be used in the same
-way as SOL, by protocols or recipients that support it.
+#### Step 2: Sending your SOL to a staking protocol
+Once your SOL is deposited, the Treasury Controller program will send your SOL from the Treasury Account to non-custodial liquid staking protocols such as Marinade Finance (Marinade) and SolBlaze.                                        
 
-For example, if you are asked to pay a recipient in SOL, you can pay them in gSOL instead.
-The recipient can then choose to either keep the gSOL, or unstake it and receive SOL.
+Treasury Controller will stake your SOL in:
+1. Marinade's stake pool 
+2. Solblaze's stake pool - called BlazeStake 
+3. Marinade's liquidity pool 
 
-Keeping SOL staked in Sunrise as long as possible maximises the climate impact of each stake
-as well as the stability of the Solana blockchain.
+_NOTE_: Sunrise Stake has future plans to integrate other non-custodial staking protocols (in addition to Marinade and SolBlaze).
+ 
+##### Stake Pools 
+When Treasury Controller stakes your SOL in Marinade's stake pool, Marinade mints an amount of msOL tokens (Marinade Staked SOL tokens) representing the size of your holding of the stake pool, which is redeemable for SOL.
 
-### I am interested in integrating gSOL into my project, how can I do that?
+For example, if Marinade's stake pool contains 20 SOL and the total minted mSOL is 10, then if you add 1 SOl, Marinade mints something like (1/20) * 10 = 0.5 mSOL. 
+The stake pool size is now 21, you hold 0.5 out of a total 10.5 mSOL, so you own (0.5 / 10.5) of the pool of 21 sol, which equals 1 SOL.
 
-gSOL is a standard SPL token, and can be integrated into any project that supports SPL tokens.
+The staking process for SolBlaze is the same as above; when the Treasury Controller program stakes your SOL into BlazeStake's stake pool, Sunrise receives bSOL tokens (BlazeStake Staked SOL tokens) relative to your SOL's size of the stake pool. Just like mSOL, bSOL is redeemable for SOL.
 
-The gSOL token address is `gso12BFMxXD7RJaTUFfZZyuLW32M1hA9LoLZN7u9Und`.
+mSOL and bSOL accrue value 
 
-You can convert the gSOL back to SOL either by unstaking it manually
-in the [Sunrise Stake app](https://app.sunrisestake.com), through the upcoming JS library, or directly
-via the protocol itself.
+##### Liquidity Pool 
+2.  A  mSOL/SOL unstake pool - through which Sunrise becomes a liquidity provider to Marinade
```

Comment:
We should mention here why we do this. We do this to ensure a certain amount of SOL is always liquidly available, (LP unstake is free) while still earning yield on it.